plugins/inputs/execd/execd.go

```diff
@@ -140,6 +141,10 @@ func (e *Execd) cmdReadOutStream(out io.Reader) {
 			if err == influx.EOF {
 				break // stream ended
 			}
+			if errors.Is(err, os.ErrClosed) {
+				e.Log.Debugf("stream closed: %w", err)
```

Comment:
`%w` is only correct for `fmt.Errorf`; also I'd indicate `ignoring` in the message so nobody gets concerned